.github/workflows/update.yaml

```diff
@@ -0,0 +1,30 @@
+name: Check for updates
+on:
+  schedule: # for scheduling to work this file must be in the default branch
+  - cron: "0 * * * *" # run every hour
+  workflow_dispatch: # can be manually dispatched under GitHub's "Actions" tab
+
+jobs:
+  flatpak-external-data-checker:
+    runs-on: ubuntu-latest
```

Comment:
Why ubuntu, shouldn't this be fedora?

Reply:
Because Ubuntu is the only supported distribution in GitHub Actions.

Anyway the distribution doesn't matter, since basically the whole workflow (except the initial git clone) is run inside of a container.